Host a Static Website with Hugo and NGINX.md

```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
 
     pkill hugo
 
-Then show the list of hugo processes.
-
-    ps aux | grep hugo
-
-Which should return output:
+Now `ps aux | grep hugo` should show no hugo servers running:
 
     username 16628  0.0  0.1  14224  1020 pts/1    S+   15:29   0:00 grep --color=auto hugo
 
 ### Web Browser Returns Bad Gateway
 
-Bad Gateway can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
+"Bad Gateway" can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
```

Comment:
One more thing: "server is not running _on_ port 1313

Reply:
I am using `at port 1313` numerous places throughout the doc. So I'll correct this issue.